docs/core/idioms/flow-to-flow.md

```diff
@@ -85,10 +85,10 @@ weekday_schedule = CronSchedule(
 
 
 # assumes you have registered the following flows in a project named "examples"
-flow_a = StartFlowRun(flow_name="A", project_name="examples", wait=True)
-flow_b = StartFlowRun(flow_name="B", project_name="examples", wait=True)
-flow_c = StartFlowRun(flow_name="C", project_name="examples", wait=True)
-flow_d = StartFlowRun(flow_name="D", project_name="examples", wait=True)
+flow_a = create_flow_run(flow_name="A", project_name="examples", wait=True)
+flow_b = create_flow_run(flow_name="B", project_name="examples", wait=True)
+flow_c = create_flow_run(flow_name="C", project_name="examples", wait=True)
+flow_d = create_flow_run(flow_name="D", project_name="examples", wait=True)
```

Comment:
There's a separate `wait_for_flow_run` task, there's no `wait` setting for `create_flow_run`

Reply:
Ah right!
In this example, should we just show how to use `create_flow_run`, or should we also explain how `wait_for_flow_run`?

Reply:
I think if updating the examples we should probably include `wait_for_flow_run`